src/Components/Patient/SampleTest.tsx

```diff
@@ -131,21 +175,33 @@ export const SampleTest = (props: any) => {
         has_sari: state.form.has_sari,
         is_unusual_course: state.form.is_unusual_course,
         is_atypical_presentation: state.form.is_atypical_presentation,
-        atypical_presentation: state.form.is_atypical_presentation ? state.form.atypical_presentation : undefined,
+        atypical_presentation: state.form.is_atypical_presentation
+          ? state.form.atypical_presentation
+          : undefined,
         diagnosis: state.form.diagnosis ? state.form.diagnosis : undefined,
-        diff_diagnosis: state.form.diff_diagnosis ? state.form.diff_diagnosis : undefined,
-        doctor_name: state.form.doctor_name ? state.form.doctor_name : undefined,
-        etiology_identified: state.form.etiology_identified ? state.form.etiology_identified : undefined,
+        diff_diagnosis: state.form.diff_diagnosis
+          ? state.form.diff_diagnosis
+          : undefined,
+        testing_facility: state.form.testing_facility,
+        doctor_name: state.form.doctor_name
```

Comment:
Codacy found an issue: [Identifier 'doctor_name' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)